src/drivers/mpu9250/mpu9250.cpp

```diff
@@ -328,7 +328,9 @@ MPU9250::init()
 #endif
 
 	/* do CDev init for the mag device node, keep it optional */
```

Comment:
@bkueng - do you think we will end up with the same issue that required [this](https://github.com/PX4/Firmware/commit/f746141afe3499ae56e75b3db06409381ead2f4b)